scripts/apps/workspace/helpers/getLabelForFieldId.js

```diff
@@ -1,32 +1,31 @@
 import {GET_LABEL_MAP} from '../content/constants';
```

Comment:
The method is called `getLabelNameResolver`, but as I can see it now not only gets names of article field labels, but all kinds of vocabularies. It'd be useful to rename it to reflect that.

Reply:
I disagree, but you go ahead with the merge and I'll rename it later.

Reply:
I think we have to rename it when we use it for something else than to get the label for a field.